src/Tokenizer/Tokens.php

```diff
@@ -287,6 +287,7 @@ public static function getBlockEdgeDefinitions(): array
      *
      * @param int $size
      */
+    #[\ReturnTypeWillChange]
```

Comment:
the more of it we will have, the easier it will be for us if we have a todo with description when to drop this and apply the typing change in method header

Reply:
For me todo is redundant here, we can prepare major upgrade checklist and put "review all #[ReturnTypeWillChange]" there 🙂.

Reply:
easier for me to grep for `@TODO` than for `@TODO|ReturnTypeWillChange|sthElseIAlreadyForgotWhatWasIt`